docker/docker-compose.yml

```diff
@@ -187,6 +187,44 @@ services:
     networks:
       - mydefault
 
+  rdf-fingerprinter-api:
+    container_name: rdf-fingerprinter-api
+    image: meaningfy/rdf-fingerprinter-api:latest
+    ports:
+      - ${RDF_FINGERPRINTER_API_PORT}:${RDF_FINGERPRINTER_API_PORT}
+    env_file: .env
+    restart: always
+    networks:
+      - mydefault
+
+  rdf-fingerprinter-ui:
+    container_name: rdf-fingerprinter-ui
+    image: meaningfy/rdf-fingerprinter-ui:latest
+    ports:
+      - ${RDF_FINGERPRINTER_UI_PORT}:${RDF_FINGERPRINTER_UI_PORT}
+    env_file: .env
+    depends_on:
+      - "rdf-fingerprinter-api"
+    restart: always
+    networks:
+      - mydefault
+
+  rdf-fingerprinter-fuseki:
+    container_name: fuseki-fingerprinter
+    image: meaningfy/fuseki:latest
+    env_file:
+      - .env
+    environment:
+      - ADMIN_PASSWORD=${RDF_FINGERPRINTER_FUSEKI_ADMIN_PASSWORD}
+      - JVM_ARGS=${RDF_FINGERPRINTER_FUSEKI_JVM_ARGS}
+    ports:
+      - ${RDF_FINGERPRINTER_FUSEKI_PORT}:3030
+    volumes:
+      - ${PWD}/${RDF_FINGERPRINTER_FUSEKI_DATA_FOLDER}/fuseki:/fuseki
```

Comment:
just realised that all the Fuseki data folders must be moved into dedicated volumes defined in volumes section